src/components/varsom-landslide-observation/varsom-landslide-observation.tsx

```diff
@@ -48,20 +48,31 @@ export class VarsomLandslideObservation {
 
 <div class="content">
 
-    {(this.DtLandSlideTime && !this.DtLandSlideTimeEnd)  ? 
+    {(this.DtLandSlideTime && (this.DtLandSlideTimeEnd != this.DtLandSlideTime))  ? 
     <varsom-key-value
     _key={this.strings && !this.shortVersion ? this.strings.Observations.LandslideObs.Time : (this.shortVersion ? null : "Tid") }
-    _value={this.DtLandSlideTime}
+    _value={new Date(this.DtLandSlideTime).toLocaleString("no", 
+    { 
+  dateStyle: "long",
+  timeStyle: "short"}
+  )}
     ></varsom-key-value>
     :""}
 
     {(this.DtLandSlideTime && this.DtLandSlideTimeEnd && this.strings)  ? 
     <varsom-key-value
     shortVersion={this.shortVersion ? this.shortVersion : null }
     _key={this.strings && !this.shortVersion ? this.strings.Observations.LandslideObs.Time : (this.shortVersion ? null : "Tid") }
-    _value={(this.strings ? this.strings.Observations.LandslideObs.Between : "Mellom") + " " + this.DtLandSlideTime
+    _value={(this.strings ? this.strings.Observations.LandslideObs.Between : "Mellom") + " " + new Date(this.DtLandSlideTime).toLocaleString("no", {
+      dateStyle: "long",
+      timeStyle: "short"  
+    })
    + " " + (this.strings ? this.strings.Observations.LandslideObs.And + " " : "og ") + 
-   this.DtLandSlideTimeEnd
+   new Date(this.DtLandSlideTimeEnd).toLocaleString("no", {
```

Comment:
I stedet for å alltid bruke "no", bør språk settes fra en global variabel. Feks engelsk bør jo "en" brukes. Men det kan eventuelt løses i en egen oppgave hvis du ikke vil løse det i denne branchen her.